x-pack/plugins/lens/kibana.json

```diff
@@ -25,6 +25,7 @@
     "eventAnnotation"
   ],
   "optionalPlugins": [
+    "expressionXY",
```

Comment:
So I am not the expert on plugins structures so it's more curiosity question, but how this is optional and the gauge and heatmap no?

Reply:
Possibly, that makes sense to make the xy required and then export icons from xy to Lens, but I don't like the idea of a stand-alone plugin being connected to the other plugin, which is representing some expression, which can be changed or removed in the future.

`optionalPlugins` can contain plugins, from which types are used only or some conditional imports.

What do you think, does it make sense to export images (I was afraid it will increase the size of the `xy` bundle. It is already big enough)?

Reply:
Your explanation makes sense for me and you have two important points here why leaving it optional 👍🏼